app/services/irs_attempts_api/event.rb

```diff
@@ -0,0 +1,65 @@
+module IrsAttemptsApi
+  Event = Struct.new(:jti, :iat, :event_type, :encrypted_event_data, keyword_init: true) do
+
+    def self.build(event_type:, session_id:, occurred_at:, event_metadata:)
+      jti = SecureRandom.uuid()
+      iat = Time.now.to_i
```

Comment:
1. I like to always inject `Time.now` so that we can have clearer tests if needed
2. let's drop the optional parens on `uuid`?

```suggestion
    def self.build(event_type:, session_id:, occurred_at:, event_metadata:, now: Time.zone.now)
      jti = SecureRandom.uuid()
      iat = now.to_i
```

Reply:
willfix and willfix